coderd/database/modelmethods.go

```diff
@@ -121,8 +134,26 @@ func (u User) UserDataRBACObject() rbac.Object {
 	return rbac.ResourceUser.WithID(u.ID).WithOwner(u.ID.String())
 }
 
-func (License) RBACObject() rbac.Object {
-	return rbac.ResourceLicense
+func (u GetUsersRow) RBACObject() rbac.Object {
+	return rbac.ResourceUser.WithID(u.ID)
+}
+
+func (u GitSSHKey) RBACObject() rbac.Object {
+	return rbac.ResourceUserData.WithID(u.UserID).WithOwner(u.UserID.String())
+}
+
+func (u GitAuthLink) RBACObject() rbac.Object {
+	// I assume UserData is ok?
+	return rbac.ResourceUserData.WithID(u.UserID).WithOwner(u.UserID.String())
+}
+
+func (u UserLink) RBACObject() rbac.Object {
+	// I assume UserData is ok?
+	return rbac.ResourceUserData.WithOwner(u.UserID.String()).WithID(u.UserID)
+}
+
+func (l License) RBACObject() rbac.Object {
+	return rbac.ResourceLicense.WithIDString(strconv.FormatInt(int64(l.ID), 10))
```

Comment:
are we using the `ID` or `UUID`?  I'm not clear what's going on with licenses in this regard, and it's an example of why we should only have one identifier.

Reply:
Honestly I was not 100% sure, but not all Licenses had a `uuid`. It is a nullable field. I was adding the `ID` in the rbac mainly for logging reasons since the the RBAC input is logged.